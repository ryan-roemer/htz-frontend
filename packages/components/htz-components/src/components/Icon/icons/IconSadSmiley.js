/* *************************************************************** *
 * THIS IS AN AUTO GENERATED FILE. PLEASE DO NOT EDIT IT DIRECTLY.
 *
 * If you want to change this Icon component , it is generated
 * from the `iconTamplate.js` file the parent directory.
 * *************************************************************** */
import React from 'react';
import { FelaComponent, } from 'react-fela';
import iconStyle from '../iconStyle';
import { iconPropTypes, iconDefaultProps, } from '../iconPropTypes';

const IconSadSmiley = ({ size, fill, color, attrs, miscStyles, ...props }) => (
  <FelaComponent
    size={size}
    fill={fill}
    color={color}
    attrs={attrs}
    miscStyles={miscStyles}
    style={iconStyle}>{({ className, }) => (
      <svg width="1em" height="1em" viewBox="0 0 256 256" className={className} {...props} {...attrs} >  <path fill="currentColor" d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z" />  <circle cx={128} cy={128} r={88} fill="none" stroke="currentColor" strokeMiterlimit={10} strokeWidth={9} /></svg>
    )}</FelaComponent>
);

IconSadSmiley.propTypes = iconPropTypes;
IconSadSmiley.defaultProps = iconDefaultProps;

export default IconSadSmiley;
