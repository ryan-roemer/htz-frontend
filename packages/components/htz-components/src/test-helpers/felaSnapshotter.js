import React from 'react';
import createRenderer from '../components/StyleProvider/createRenderer';
import StyleProvider from '../components/StyleProvider/StyleProvider';
import createFelaSnapshotter from './createFelaSnapshotter';

const testRenderer = createRenderer({ isRtl: true, });
const felaSnapshotter = createFelaSnapshotter(StyleProvider, testRenderer);

export default felaSnapshotter;
