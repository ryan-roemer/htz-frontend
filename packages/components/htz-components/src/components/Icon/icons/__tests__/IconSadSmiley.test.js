
/* global window */
/* *************************************************************** *
 * THIS IS AN AUTO GENERATED FILE. PLEASE DO NOT EDIT IT DIRECTLY.
 *
 * If you want to change the styleguide example, it is generated
 * from the `embedsFileTemplate.js` file is this directory.
 * *************************************************************** */
import React from 'react';
import felaSnapshotter from '../../../../test-helpers/felaSnapshotter';
import IconSadSmiley from '../IconSadSmiley';

describe('<IconSadSmiley />', () => {
  /* The error tests are left as documentation, but are commented out for the redability of the test results */
  // describe('Errors', () => {
  //   it('throw an error when the value passed to the "size" prop isn't a number', () => {
  //     expect(() => felaSnapshotter(<IconSadSmiley size="2rem" />)).toThrow(
  //       /^An Icon's "size" prop may only be passed a "number"/
  //     );
  //   });
  //   it('throw an error when the value passed to the "color" prop isn't a named color (string)', () => {
  //     expect(() => felaSnapshotter(<IconSadSmiley color="red" />)).toThrow(
  //       /is not a named-color/
  //     );
  //   });
  //   it('throw an error when the value passed to the "color" prop isn't a named color (array of strings)', () => {
  //     expect(() =>
  //       felaSnapshotter(<IconSadSmiley color={[ 'primary', 'nope', ]} />)
  //     ).toThrow(/is not a named-color/);
  //   });
  //   it('throw an error when the value passed to the "fill" prop isn't a named color (string)', () => {
  //     expect(() => felaSnapshotter(<IconSadSmiley fill="red" />)).toThrow(
  //       /is not a named-color/
  //     );
  //   });
  //   it('throw an error when the value passed to the "fill" prop isn't a named color (array of strings)', () => {
  //     expect(() =>
  //       felaSnapshotter(<IconSadSmiley fill={[ 'primary', 'nope', ]} />)
  //     ).toThrow(/is not a named-color/);
  //   });
  // });
  describe('DOM element', () => {
    testCase('render correctly when no props are passed', <IconSadSmiley />);
    testCase(
      'pass attributes defined in the "attrs" prop to the DOM element',
      <IconSadSmiley attrs={{ 'aria-hidden': true, }} />
    );
    testCase('add "id" attribute to DOM element', <IconSadSmiley id="hasId" />);
    testCase(
      'add "onClick" attribute to DOM element',
      <IconSadSmiley id={e => console.log('a')} />
    );
  });

  describe('styles', () => {
    testCase("set icon's color", <IconSadSmiley color={[ 'primary', 'base', ]} />);
    testCase(
      "set icon's color responsively",
      <IconSadSmiley
        color={[
          { until: 'xl', value: [ 'primary', 'base', ], },
          { from: 'xl', value: [ 'bodyText', 'base', ], },
        ]}
      />
    );
    testCase("set icon's fill", <IconSadSmiley fill={[ 'primary', 'base', ]} />);
    testCase(
      "set icon's fill responsively",
      <IconSadSmiley
        fill={[
          { until: 'xl', value: [ 'primary', 'base', ], },
          { from: 'xl', value: [ 'bodyText', 'base', ], },
        ]}
      />
    );
    testCase("set icon's size", <IconSadSmiley size={6} />);
    testCase(
      "set icon's size responsively",
      <IconSadSmiley
        size={[ { until: 'xl', value: 6, }, { from: 'xl', value: 7, }, ]}
      />
    );
    testCase(
      'set misc styles on an icon',
      <IconSadSmiley miscStyles={{ position: 'relative', }} />
    );
    testCase(
      'responsively set misc styles on an icon',
      <IconSadSmiley
        miscStyles={{
          position: [
            { until: 'xl', value: 'relative', },
            { from: 'xl', value: 'static', },
          ],
        }}
      />
    );
  });
});

function testCase(name, Component, { only = false, } = {}) {
  const testFunc = only ? it.only : it;
  testFunc(name, () => {
    const { component, styles, } = felaSnapshotter(Component);
    expect(component).toMatchSnapshot();
    expect(styles).toMatchSnapshot();
  });
}
