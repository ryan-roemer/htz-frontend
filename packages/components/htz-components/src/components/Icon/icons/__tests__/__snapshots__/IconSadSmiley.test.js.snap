// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconSadSmiley /> DOM element add "id" attribute to DOM element 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    id="hasId"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> DOM element add "id" attribute to DOM element 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}"`;

exports[`<IconSadSmiley /> DOM element add "onClick" attribute to DOM element 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    id={[Function]}
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> DOM element add "onClick" attribute to DOM element 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}"`;

exports[`<IconSadSmiley /> DOM element pass attributes defined in the "attrs" prop to the DOM element 1`] = `
<div>
  <svg
    aria-hidden={true}
    className="htz-a htz-b htz-c"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> DOM element pass attributes defined in the "attrs" prop to the DOM element 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}"`;

exports[`<IconSadSmiley /> DOM element render correctly when no props are passed 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> DOM element render correctly when no props are passed 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}"`;

exports[`<IconSadSmiley /> styles responsively set misc styles on an icon 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d htz-e"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> styles responsively set misc styles on an icon 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}@media (max-width: 79.9375em){.htz-d{position:relative}}@media (min-width: 80em){.htz-e{position:static}}"`;

exports[`<IconSadSmiley /> styles set icon's color 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> styles set icon's color 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:#0B7EB5}.htz-c{fill:#FFF}"`;

exports[`<IconSadSmiley /> styles set icon's color responsively 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> styles set icon's color responsively 2`] = `".htz-a{vertical-align:-0.15em}.htz-d{fill:#FFF}@media (max-width: 79.9375em){.htz-b{color:#0B7EB5}}@media (min-width: 80em){.htz-c{color:#2D2D2D}}"`;

exports[`<IconSadSmiley /> styles set icon's fill 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> styles set icon's fill 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#0B7EB5}"`;

exports[`<IconSadSmiley /> styles set icon's fill responsively 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> styles set icon's fill responsively 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}@media (max-width: 79.9375em){.htz-c{fill:#0B7EB5}}@media (min-width: 80em){.htz-d{fill:#2D2D2D}}"`;

exports[`<IconSadSmiley /> styles set icon's size 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> styles set icon's size 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}.htz-d{font-size:6rem}"`;

exports[`<IconSadSmiley /> styles set icon's size responsively 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d htz-e"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> styles set icon's size responsively 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}@media (max-width: 79.9375em){.htz-d{font-size:6rem}}@media (min-width: 80em){.htz-e{font-size:7rem}}"`;

exports[`<IconSadSmiley /> styles set misc styles on an icon 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d"
    height="1em"
    viewBox="0 0 256 256"
    width="1em"
  >
      
    <path
      d="M115 104c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm46 1c0 5.5-4.5 10-10 10s-10-4.5-10-10 4.5-10 10-10 10 4.5 10 10zm1.6 55.7c1.8-1.7 1.9-4.6 0-6.4A48.2 48.2 0 0 0 128 139a48 48 0 0 0-34.7 15.3 4.5 4.5 0 0 0 0 6.4 4.6 4.6 0 0 0 6.5-.1c7.9-8.1 17.8-12.6 28.2-12.6s20.4 4.5 28.2 12.6c.9.9 2 1.4 3.2 1.4 1.2-.1 2.4-.5 3.2-1.3z"
      fill="currentColor"
    />
      
    <circle
      cx={128}
      cy={128}
      fill="none"
      r={88}
      stroke="currentColor"
      strokeMiterlimit={10}
      strokeWidth={9}
    />
  </svg>
</div>
`;

exports[`<IconSadSmiley /> styles set misc styles on an icon 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}.htz-d{position:relative}"`;
